core/local/atom/add_infos.js

```diff
@@ -65,7 +71,10 @@ function loop(
             // If kind is unknown, we say it's a file arbitrary
             if (event.kind !== 'directory' && event.kind !== 'file') {
               _.set(event, [STEP_NAME, 'kindConvertedFrom'], event.kind)
-              event.kind = 'file'
+
+              const doc =
+                event._id && (await opts.pouch.byIdMaybeAsync(event._id))
```

Comment:
Can it fail? (it == `await opts.pouch.byIdMaybeAsync(event._id)`)

Reply:
And, if it can fail, what happens?

Reply:
It can fail if we don't have any document with this `_id` in Pouch.
If it fails, `doc` will be `undefined` and we'll resort back to forcing the event `kind` to `file` as I don't see any other solution for now.